src/plugins/chart_expressions/expression_metric/common/expression_functions/metric_vis_function.ts

```diff
@@ -84,6 +92,44 @@ export const metricVisFunction = (): MetricVisExpressionFunctionDefinition => ({
       throw new Error('Palette must be provided when using percentageMode');
     }
 
+    // currently we can allow colorize full container only for one metric
+    if (args.colorFullBackground) {
+      if (args.bucket) {
+        throw new Error(
```

Comment:
Can you, please, move the error message translation out of the function scope. Mostly for the consistency and reducing amount of the code, which is not describing the logic.

Example: https://github.com/elastic/kibana/blob/8f4de1b8a8c6de2041638ea06928fc420c981919/src/plugins/chart_expressions/expression_partition_vis/common/expression_functions/mosaic_vis_function.ts#L94